server/index.js

```diff
@@ -2,8 +2,116 @@ const express = require('express');
 const app = express();
 const port = 4000;
 
+const db = require('../database/connect.js');
+
 app.get('/', (req, res) => {
-  res.send('Hello World!')
+  res.send('Hello World');
+});
+
+// Retrieves the list of products.
+app.get('/products', (req, res) => {
+  db.client.query('SELECT * FROM Product_Info LIMIT 10', (err, data) => {
+    if (err) {
+      // console.log('error in /products - ', err);
+      res.send(err);
+    } else {
+      // console.log('data from /products - ', data.rows);
+      res.send(data.rows);
+    }
+  })
+});
+
+// Returns all product level information for a specified product id.
+app.get('/products/:product_id', (req, res) => {
+  let product_id = 1;
```

Comment:
Is this is a placeholder? Trying to understand why you're declaring this vs extracting it from the request

Reply:
Maybe a visual test to see if it's getting the proper data back?

Reply:
At the moment is it a placeholder, since we don't have a client connected yet, I don't have anywhere to grab the actual product_id requested. As we progress, I anticipate this would be updated to something coming from the client.